tg_bot/modules/sql/disable_sql.py

```diff
@@ -57,7 +57,7 @@ def enable_command(chat_id, enable):
 
 
 def is_command_disabled(chat_id, cmd):
-    return cmd in DISABLED.get(str(chat_id), set())
+    return str(cmd).lower() in DISABLED.get(str(chat_id), set())
```

Comment:
Might be worth making sure the `DISABLED` var gets initialised with lowercase commands too, to preserve backwards compatibility.

Reply:
So when commands are disabled, lowercase them there then add to db?

Reply:
You don't need to add them to the db, just make sure that when you populate the DISABLED var, you lowercase everything.
Also, you want to remove the str() here too.